src/index_runner/es_indexers/sample_set.py

```diff
@@ -17,6 +17,25 @@
 # _VER_SAMPLE_INDEX_NAME = 'sample_version_' + str(_SAMPLE_INDEX_VERSION)
 
 
+def _get_document(sample_document_id):
```

Comment:
I wonder if this should be a generic function inside `es_utils.py`. There is a somewhat similar function in there already

Reply:
moved to es_utils and made more generic